packages/editor/src/pages/Editor/components/Backstage/MySolutions/index.tsx

```diff
@@ -85,16 +85,18 @@ class MySolutions extends React.Component<IProps> {
                 return true;
               }
 
-              const megastring = [
-                solution.id,
+              const megaString = [
+                process.env.NODE_ENV === 'production'
+                  ? null
+                  : solution.id /* For Cypress test framework, need to include the solution ID so can search based on it */,
                 solution.name,
                 solution.description,
-                ...solution.files.map(file => file.content),
               ]
                 .filter(Boolean)
-                .join(' ');
+                .join(' ')
+                .toLowerCase();
 
-              return megastring.includes(this.state.filterQuery);
+              return megaString.includes(this.state.filterQuery.toLowerCase());
```

Comment:
if you want case insensitive search, have setFilterQuery set the state to already be lower case that way it doesn't need to get recomputed every time / called in each place filterQuery is used